crates/uv-settings/src/settings.rs

```diff
@@ -456,6 +459,21 @@ pub struct ResolverInstallerOptions {
         possible_values = true
     )]
     pub prerelease: Option<PrereleaseMode>,
+    /// The strategy to use when selecting multiple versions of a given package across Python
+    /// versions and platforms.
+    ///
+    /// By default, uv will minimize the number of versions selected for each package (`fewest`),
+    /// to minimize differences between environments. Under `latest`, uv will select the latest
+    /// compatible version for each environment, even if it results in more versions being selected.
+    #[option(
+        default = "\"fewest\"",
+        value_type = "str",
+        example = r#"
+            resolution = "latest"
```

Comment:
sorry for the drive by comment, but should this be `multi-version-mode = "latest"`? `settings.md` uses `resolution` in the examples too

Reply:
Thank you yeah, this is wrong.